### ISSUE TITLE
How to use Git with a mixed Python / C++ solution

### ISSUE THREAD

**Issue by goofy2k** [MicrosoftDocs/visualstudio-docs#10063]
I created a mixed Python / C++ solution. 
When I create a repo, the IDE tells me "Your solution contains files outside the solution folder which will not be managed by Git". On inspection these are  C++ related files.
How can I ensure that local changes in the C++ code are also synced with a repo?

In the docs I found a section [Work with multiple repos](https://learn.microsoft.com/en-us/visualstudio/version-control/git-multi-repository-support?view=vs-2022) but I have no clue if this relates to my issue.

How can I do source control on files in my solution that are missing in the repo that I created?
Can I couple syncing of the different Projects in my Solution ?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b430664f-79e1-35fa-5292-34f517092b08
* Version Independent ID: c72b1131-9035-bb27-2465-50328f12dc93
* Content: [The Git experience in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/version-control/git-with-visual-studio?view=vs-2022)
* Content Source: [docs/version-control/git-with-visual-studio.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/version-control/git-with-visual-studio.md)
* Service: **visual-studio-windows**
* Sub-service: **general-ide**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by WilliamAntonRohm** [comment]
@goofy2k -- Goofy, please consider these resources:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@ghogen -- Gordon, please look into this potential documentation issue.

**Comment by goofy2k** [comment]
Thanks! I somehow got it working, but I am afraid that I can not reproduce
it in a next occasion. It would be great indeed if a few words could be
spent on this in the docs.

Fred

Op ma 26 feb. 2024 16:42 schreef William Anton Rohm <
***@***.***>:

**Comment by goofy2k** [comment]
So I got it working somehow before.  I had to install Visual Studio on a different machine. Most of my toolbox is working now but, again, I have trouble getting source control right. The docs are not very helpfull in this.

My problem:  I have a solution consisting of 2 projects:  a Python one and a C++ one.  For the Python project I can sync changes with my repo on Github. The C++ project files are in different folders. I don't see a way to create a repo for these files.
